File: elixir/src/hello_loop.exs
world_list = ["a", "b", "c"]

for world <- world_list do
  IO.puts("world is #{world}!")
end

world_list
|> Enum.map(fn world ->
  IO.puts("world is #{world}!")
end)

world_map = %{a: "aaa", b: "bbb", c: "ccc"}

for {key, value} <- world_map do
  IO.puts("world #{key} is #{value}!")
end

world_map
|> Enum.map(fn {key, value} ->
  IO.puts("world #{key} is #{value}!")
end)

File: rust/src/hello_loop.rs
use std::collections::HashMap;

fn main() {
    let world_list = ["a", "b", "c"];

    for world in world_list.iter() {
        println!("world is {}!", world);
    }

    let mut world_map = HashMap::new();
    world_map.insert("a", "aaa");
    world_map.insert("b", "bbb");
    world_map.insert("c", "ccc");

    for (key, value) in world_map.into_iter() {
        println!("world {} is {}!", key, value)
    }
}
